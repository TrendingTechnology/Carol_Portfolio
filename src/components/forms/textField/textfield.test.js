import React from 'react';
import { render, screen } from '../../../infra/test/testUtils';
import TextField from './textfield';

describe('<TextField />', () => {
  test('renders component', () => {
    render(
      <TextField
        placeholder="Nome"
        value="Carol"
        onChange={() => { }}
        name="nome"
        label="Nome"
      />,
    );

    const label = screen.getByText(/nome/i);
    const textField = screen.getByPlaceholderText(/nome/i);

    expect(textField).toMatchSnapshot();
    expect(label).toMatchSnapshot();
  });
});
