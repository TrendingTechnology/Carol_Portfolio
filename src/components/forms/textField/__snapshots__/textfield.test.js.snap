// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextField /> renders component 1`] = `
.c0 {
  font-size: 16px;
  font-weight: 400;
  line-height: 1;
}

.c1 {
  width: 100%;
  border: none;
  border-radius: 5px;
  padding: 10px 5px;
  outline: 0;
  background-color: #E0E0E0;
  margin-bottom: 2px;
}

<input
  class="c0 c1"
  name="nome"
  placeholder="Nome"
  required=""
  value="Carol"
/>
`;

exports[`<TextField /> renders component 2`] = `
<label
  aria-label="nome"
  for="nome"
>
  Nome
</label>
`;
